apps/src/calc/expressionNode.js

```diff
@@ -166,7 +166,11 @@ ExpressionNode.prototype.evaluate = function (globalMapping, localMapping) {
       // We're calling a new function, so it gets a new local scope.
       var newLocalMapping = {};
       functionDef.variables.forEach(function (variable, index) {
-        var childVal = this.getChildValue(index);
+        var evaluation = this.children_[index].evaluate(globalMapping, localMapping);
```

Comment:
Yes. In the case of f(1 + 1), get childValue() returns '+', whereas evaluating gets us 2.